snapcraft.yaml

```diff
@@ -0,0 +1,41 @@
+name: subiquity
+version: "0.0.32"
+summary: Ubuntu installer
+description: The Ubuntu server installer
+confinement: classic
+
+apps:
+  subiquity:
+    command: usr/bin/subiquity
+  console-conf:
+    command: usr/bin/console-conf
+  probert:
+    command: bin/probert
+
+parts:
+  subiquity:
+    plugin: python
+    build-packages: [python-setuptools]
+    stage-packages: [curtin]
+    python-packages:
+      - urwid
+      - pyyaml
+      - pyudev
+      - attrs
+    source: https://github.com/CanonicalLtd/subiquity.git
```

Comment:
Currently, it works better having it set to the real location of the source code for building things. When merged and integrated into LP for daily building, we'll want to revisit (as LP probably can't reach it), but it also apparently can't do snap recipes from purely git trees.